src/CHANGES.txt

```diff
@@ -17,6 +17,11 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       the index from find() was not used.
     - Turn previously deprecated debug options into failures:
       --debug=tree, --debug=dtree, --debug=stree, --debug=nomemoizer.
+    - If SCONS_CACHE_MSVC_CONFIG shell environment variable is set,
```

Comment:
Can you add "Added EXPERIMENTAL new logic to cache MSVC config. Use with Caution."